### PR TITLE
update WISE logo and info

### PR DESCRIPTION
**Pull request by coffee-converter**
Updates the WISE logo and info.

### CODE REVIEW

**Review by coffee-converter — COMMENTED** [review]
Curious why the info.json deletion?

**Comment by Cryptocool1** [comment]
> Curious why the deletion?

it was unnecessary file

**Comment by Cryptocool1** [comment]
the correct one is blockchains/smartchain/assets/0xbfBEe3dAc982148aC793161f7362344925506903/info.json

**Comment by coffee-converter** [comment]
> > Curious why the deletion?
> 
> it was unnecessary file info.json with socials

I removed the socials in one of the commits

**Comment by coffee-converter** [comment]
> 0xbfBEe3dAc982148aC793161f7362344925506903

I have no idea what that is, it's not this WISE token contract.

**Comment by Cryptocool1** [comment]
> > 0xbfBEe3dAc982148aC793161f7362344925506903
> 
> I have no idea what that is, it's not this WISE token contract.

it's file in your PR

**Comment by coffee-converter** [comment]
> > > 0xbfBEe3dAc982148aC793161f7362344925506903
> > 
> > 
> > I have no idea what that is, it's not this WISE token contract.
> 
> it's file in your PR

It was first included in a commit by the merge bot, I guess the bot sometimes includes fixes to other files in other chains?  Weird.

https://github.com/trustwallet/assets/pull/7991/commits/55647356ef8f6afda13fdc36a8f6350d2188b724

**Comment by coffee-converter** [comment]
@Cryptocool1 but the problem remains, in my PR you deleted the WISE Token info.json which is the correct one here.

https://github.com/trustwallet/assets/pull/7991/commits/9dc210af665a9d3642d0c1cb0f81a04daa0a13fb

**Comment by coffee-converter** [comment]
@Cryptocool1 see on master, after the merge my token has missing info.json:

https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6

File: blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json [hunk 1]
```diff
@@ -1,11 +0,0 @@
-{
```
Comment (coffee-converter) [inline blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json:1]: @Cryptocool1 please put this file back and merge again, this is my token's correct info.json and you deleted it incorrectly.

Comment (coffee-converter) [reply blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json:1]: I paid 250 TWT to update my logo and info, and you've deleted the info.  Please correct this ASAP.

Comment (coffee-converter) [reply blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json:1]: @catenocrypt @zachzwei can someone please assist?

**Comment by coffee-converter** [comment]
> the correct one is blockchains/smartchain/assets/0xbfBEe3dAc982148aC793161f7362344925506903/info.json

You're incorrect.  Look at the initial request commit, it is to update my logo and info for `blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6`.  That is the correct asset for this pull request. 
 
 Please revert your commit that deletes my updated info.json, thanks!

**Comment by vm06007** [comment]
@Cryptocool1 this ain't cool at all! Our team submits the correct token info, paid the price and you've deleted our json file with all the info! Is this some kind of a joke! We want then a refund and you can revert this PR!!!!!

**Comment by vm06007** [comment]
@Cryptocool1 this ain't cool at all! Our team submits the correct token info, paid the price and you've deleted our json file with all the info! Is this some kind of a joke! We want then a refund and you can revert this PR!!!!!

File: blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json (hunk 1 above)
Comment (vm06007) [reply blockchains/ethereum/assets/0x66a0f676479Cee1d7373f3DC2e2952778BfF5bd6/info.json:1]: @Cryptocool1 this ain't cool at all! Our team submits the correct token info, paid the price and you've deleted our json file with all the info! Is this some kind of a joke! We want then a refund and you can revert this PR!!!!!

**Comment by vm06007** [comment]
> 0xbfBEe3dAc982148aC793161f7362344925506903

this is NOT our token!

**Review by vm06007 — COMMENTED** [review]
this needs to be reverted! @Cryptocool1

**Comment by vm06007** [comment]
> the correct one is blockchains/smartchain/assets/0xbfBEe3dAc982148aC793161f7362344925506903/info.json

this is not correct! the correct one is 0x66a0f676479cee1d7373f3dc2e2952778bff5bd6 !!!!

**Comment by catenocrypt** [comment]
Info file added back, https://github.com/trustwallet/assets/pull/8030/files

**Pull request merged**